meinberlin/assets/scss/_shame.scss

```diff
@@ -20,6 +20,23 @@
     @extend .rating-button--small;
 }
 
+// used by adhocracy4 for the follow button, long-term we'd like
+// our own classes.
+.btngroup .btn {
+    @extend .button;
+    @extend .button--light;
+
+    &:first-child {
+        border-bottom-right-radius: 0;
+        border-top-right-radius: 0;
+    }
+
+    &:last-child {
+        border-bottom-left-radius: 0;
+        border-top-left-radius: 0;
+    }
+}
```

Comment:
We already have `.button-group`. Maybe it is better to extend that here. (And maybe we should just rename our `.button` to `.btn` to avoid this kind of compatibility issues)